### ISSUE TITLE
Crucible not crafting 

### ISSUE THREAD

**Issue by LodtheFraud** [BrassGoggledCoders/SteamAgeRevolution#51]
In a testing world, I have set up a crucible with buildcraft pipes inputting steam and iron. These go in as usual. I have an output on the side as well. However, the objects inside do not smelt into molten metals. 
![image](https://user-images.githubusercontent.com/50425180/60480729-ad4bec80-9c58-11e9-8320-2360fafd5375.png)

If need be, I can provide more info

**Comment by FreneticScribbler** [comment]
Hmm. Do you have tinkers?

**Comment by LodtheFraud** [comment]
Yes, here is the full modlist - https://docs.google.com/document/d/1u6Hwa_uRSyyyk771THu6C2t8MZAtrMLOcMYlQIJBIm8/edit?usp=sharing

**Comment by LodtheFraud** [comment]
Update - My crucible started crafting today after I randomly broke and replaced a few blocks (No changes were made to its structure). It managed to output molten iron a few times. However, it began getting around halfway done creating iron, then stopped, and reset. This keeps happening now.

**Comment by FreneticScribbler** [comment]
Yeah, that one I can reproduce... :/

**Comment by FreneticScribbler** [comment]
Crucible output tank size was wrong...

**Comment by LemADEC** [comment]
Any chance we get a new version on Curse with those "fixed-in-dev" issues?

**Comment by FreneticScribbler** [comment]
Soon :tm: - it is easier said than done because they have been fixed as part of a major rewrite...

**Comment by LemADEC** [comment]
Post alpha for us to test then :)

**Comment by drizzs** [comment]
i say we just port this bad boy to 1.14.4 frenetic! lol

I dont think the rewrite is ready for that Lem

**Comment by FreneticScribbler** [comment]
It's not far off alpha to be fair. Just need to get a hot minute to do any work....

I'm not porting yet.

**Comment by drizzs** [comment]
> 
> 
>
> 
> yet.

**Comment by LemADEC** [comment]
stabilizing before breaking further, I like that plan
